production/helm/values.yaml

```diff
@@ -158,6 +158,30 @@ promtail:
       mountPath: /var/lib/docker/containers
       readOnly: true
 
+  readinessProbe:
+    failureThreshold: 5
+    exec:
+      command:
+      - /bin/sh
+      - -c
+      - /bin/ps -ef | grep promtail | grep -v grep
```

Comment:
A health check that does `ps | grep` is completely unnecessary in Kubernetes. Kubernetes already verifies the main process is running and restarts the container if it isn't. It would be better if you added a `/ready` endpoint in promtail code that this can call.

Reply:
use `/metrics` to check instead, it may be the least size of change.